components/dashboard/src/start/CreateWorkspace.tsx

```diff
@@ -17,6 +17,7 @@ import { openAuthorizeWindow } from "../provider-utils";
 import { SelectAccountPayload } from "@gitpod/gitpod-protocol/lib/auth";
 import { SelectAccountModal } from "../settings/SelectAccountModal";
 import { watchHeadlessLogs } from "../components/PrebuildLogs";
+import CodeText from "../components/CodeText";
```

Comment:
Interesting, I had no idea we had this component. 😅 I agree with @trumbitta that [a `<code>` element would be more semantic than a `<span>`](https://github.com/gitpod-io/gitpod/pull/7302#discussion_r781160513).

Also, creating single-line React components is not the best way to share common style (because it adds unnecessary code and complexity, and it's too easy to forget about using the component where relevant). I think we should remove the `CodeText` component again, and instead apply our common CSS rules to the `code` element inside [index.css](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/src/index.css).

However, this is off-topic here and can definitely be a follow-up issue / Pull Request. @trumbitta would you be interested in filing an issue for this? 🙏

Reply:
I had already created one :) might be worth it to chime in and add your perspective? https://github.com/gitpod-io/gitpod/issues/7555